comm/index.html

```diff
@@ -84,9 +85,85 @@
+            Besides a description of the problem, some additional information can make the difference between your issue being looked at or it being put off for a rainy day:
+            <ul>
+              <li>The version of GeoServer you are using? (a release or nightly build?)</li>
+              <li>Example request or data that reproduces the problem?</li>
+              <li>A stack trace indicating where the failure occurred?</li>
+              <li>The Java Version and operating system you are using?</li>
+            </ul>
+        </li>
+        <li>What Happens Next?
+            <p>On creation a notification will automatically be sent to the geoserver-devel list. JIRA sends notifications for everything done on the issue, to the reporter, the assignee, and to anyone who clicks on the link to ‘watch’ an issue.</p>
+            <p>This is why you must sign up for an account, so that JIRA can email you when updates are done. Your email will not be used for anything else. One nice little feature of JIRA is that if you reply to the email sent for notification, then the reply will show up as a comment on the issue.</p>
+        </li>
+        <li>For New Bugs we may need assistance to reproduce the issue before a fix can be considered.
+          <p>Usually the a developer will need to ask you for more information. Until they can reproduce you issue, or you volunteer to test, not much is going to happen.</p>
+          <p>Example: If you are on some exotic hardware (like oracle on windows server) that we do not have public access to you will probably need to arrange to meet on IRC and test out different solutions with a developer.</p>
+        </li>
+        <li>For new bugs with a pull request or patch.
+           <p>You link the pull request, or attach a code patch, to the Jira task and ask a developer to include your fix in the next release.
+           <p>For fixes: please include a JUnit test case showing that your fix does something. If a developer needs to spend 30 mins reproducing your problem in order to show your fix addresses the issue it will be put off for another day.</p>
+           <p>For improvements: You may be asked to discuss the idea on geoserver-devel on how best to include your great idea. For extensive changes a proposal may be required.</p>
+        </li>
+        <li>
+           Please keep in mind that many members of the GeoServer team have a day job. Just as you volunteer your time to report an issue they volunteer their time to support the project.
+        </li>
+      </ol>
+      <p>
+      As GeoServer is open source, everyone is encouraged to fix bugs and submit patches. Even if you are not a core developer, we welcome patches <a href="https://github.com/geoserver/geoserver/blob/master/CONTRIBUTING.md">contributed</a> through GitHub <a href="https://github.com/geoserver/geoserver/pulls">pull requests</a>, or attached to a JIRA issue.
+      </p>
+    </div>
+    <div class="alert alert-warning">
```

Comment:
Maybe push this one up? I fear we might have already lost our reader here, and this one is important.